TMessagesProj/src/main/AndroidManifest.xml

```diff
@@ -86,6 +86,7 @@
         android:restoreAnyVersion="true"
         android:backupAgent=".BackupAgent"
         android:hardwareAccelerated="@bool/useHardwareAcceleration"
+        android:label="@string/ForkGram"
```

Comment:
I suggest to use @string/AppName, that res must be defined in TMessagesProj/src/main/res/values/strings.xml while for NekogramX there is the label NekoX for this project there is AppName with value Fork Client, if you want ForkGram you must define a new pair key ForkGram value ForkGram

Reply:
Oh, that makes sense - thanks for the Info!
I guess "Fork Client" is also fine - that's probably what's displayed for the non F-Droid build as well i assume?

Reply:
Ok it seems likely that the developer intention is to have the Android app called "Fork Client" (and the desktop version Forkgram), and so that the page on F-Droid is wrong